be/src/storage/tablet.cpp

```diff
@@ -1073,6 +1073,8 @@ void Tablet::build_tablet_report_info(TTabletInfo* tablet_info) {
             // and perform state modification operations.
         }
         tablet_info->__set_version(max_version);
+        // TODO: support getting minReadableVersion
+        tablet_info->__set_min_readable_version(1);
```

Comment:
It's better to add a more concise message in the commit message. It will address a long-history problem.

Reply:
added